source/cloud/windows.md

```diff
@@ -33,33 +38,68 @@ Click or select the `Overview` tab, you will see the Windows admin password
+
+## Creating the image
+Create the disk: 
+```
+qemu-img create -f qcow2 windows.qcow2 40G
+```
+Install windows on the disk, mounting all the required drives:
+```
+virt-install --connect qemu:///system   --name windows --ram 6144 --vcpus 2   --network network=default,model=virtio --disk path=windows.qcow2,format=qcow2,device=disk,bus=virtio --cdrom Windows.iso --disk path=unattend.iso,device=cdrom --disk path=virtio-win.iso,device=cdrom   --vnc --os-variant=win10
+```
+Or `--os-variant=win11` for win11.
+
+### Windows setup
+Most windows setup options are taken care of by `unattend.iso` except the partitioning. 
+#### Drivers
+In the partitioning window, click `load drivers` and browse to `NetKVM\w10\amd64` on the virtio drive and select that folder. (Obviously select `w11` if on Windows 11), then `next`.
+
+Do the same for `viostor\w10\amd64`. 
+
+You should now have acccess to the internet and be able to see the virtual hard drive. Click `next` and let windows install. 
+
+### Configuring windows
+After some time, Windows will start. `unattend.iso` runs these actions:
+1) import the viostor drivers (not sure if necessary)
+2) enable unrestricted powershell for cloud init
+3) Download and run cloudbase cloud-init
+
+Wait for the download to complete.The cloudinit wizard will appear automaticaly. 
+
+Before continuing, make any changes to the OS you would still like to make, as those will be included in the final image. 
+When you're ready, follow the install wizard. In the `configuration options` screen, make sure to set COM1 as the logging port. Everything else can be left default.
+
+Before starting sysprep, open powershell as administrator and execute `C:\Windows\cloudconfig.ps1`.
```

Comment:
also provide an extra screenshot for this part and how to open the admin powershell from the installation page